src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -0,0 +1,393 @@
+    formRef: FormIF
+  }
+  readonly PdfPageSize = PdfPageSize
+
+  @Getter(useStore) getContinuationAuthorization!: ContinuationAuthorizationIF
+  @Getter(useStore) getCurrentDate!: string
+  @Getter(useStore) getKeycloakGuid!: string
+  @Getter(useStore) getShowErrors!: boolean
+
+  @Action(useStore) setContinuationAuthorization!: (x: ContinuationAuthorizationIF) => void
+
+  readonly INPUT_FILE_LABEL = 'Continuation authorization'
+
+  // Local properties
+  authorization = null as ContinuationAuthorizationIF
+  fileValidity = false
+  fileUploadCustomErrorMsg = ''
+  snackbar = false
+  snackbarText = ''
+
+  get authorityNameRules (): Array<VuetifyRuleFunction> {
+    return [
+      (v: string) => !!v || 'Authority Name is required'
+    ]
+  }
+
+  get authorizationDateRules (): Array<VuetifyRuleFunction> {
+    return [
+      (v) => !!v || 'Authorization Date is required',
+      () => (this.authorization.date <= this.getCurrentDate) ||
+        'Authorization Date cannot be in the future'
+    ]
+  }
+
+  get expiryDateRules (): Array<VuetifyRuleFunction> {
+    return [
+      () => !this.authorization.expiryDate ||
+        (this.authorization.expiryDate >= this.authorization.date) ||
+        'Expiry Date cannot be before Authorization Date'
+    ]
+  }
+
+  /** The number of file upload componenets to display (max 5). */
+  get numUploads (): number {
+    return Math.min((this.authorization.files.length + 1), 5)
+  }
+
+  mounted (): void {
+    // set or initialize authorization object
+    this.authorization = this.getContinuationAuthorization || { files: [] } as ContinuationAuthorizationIF
+  }
+
+  /**
+   * Called when FileUploadPreview tells us whether a file is valid.
+   * This is called right before the File Selected event.
+   */
+  onFileValidity (valid: boolean): void {
+    this.fileValidity = valid
+  }
+
+  /**
+   * Called when FileUploadPreview tells us about a new or cleared file.
+   * This is called right after the File Validity event.
+   */
+  async onFileSelected (index: number, file: File): Promise<void> {
+    // reset state of file uploader to ensure not in manual error mode
+    this.fileUploadCustomErrorMsg = ''
+
+    if (file) {
+      // verify that file is valid
+      if (!this.fileValidity) {
+        this.snackbarText = 'Invalid file. It will not be saved.'
+        this.snackbar = true
+        return // don't add to list
+      }
+
+      // verify that file doesn't already exist
+      if (this.authorization.files.find(f => f.file.name === file.name)) {
+        this.snackbarText = 'File already uploaded. It will not be saved again.'
+        this.snackbar = true
+        return
+      }
+
+      // try to upload to Minio
+      let psu: PresignedUrlIF
+      try {
+        psu = await this.getPresignedUrl(file.name)
+        const res = await this.uploadToUrl(psu.preSignedUrl, file, psu.key, this.getKeycloakGuid)
+        if (!res || res.status !== 200) throw new Error()
```

Comment:
I compressed the above from a separate method (as in other filings) to these few lines. That's all that's needed to do the actual upload!